client/src/components/landing/Login.js

```diff
@@ -0,0 +1,52 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+import { connect } from 'react-redux';
+
+import { authActions } from '../../sagas/actions';
+
+const LoginForm = ({ login }) => {
+  const [playerID, setPlayerID] = useState('');
+  return (
+    <div className='max-w-sm m-auto mt-4'>
+      <h1 className='text-4xl'>Login to Cribbage</h1>
+      <form
+        onSubmit={e => {
+          e.preventDefault();
+          login(playerID);
+        }}
+      >
+        <input
+          placeholder='Username'
+          onChange={e => setPlayerID(e.target.value)}
+          className='pl-2 h-8 shadow-sm rounded-lg block w-full focus:outline-none focus:shadow-md'
+        ></input>
+        <p className='mt-1 text-xs text-gray-600'>
+          Don't have an account?{' '}
```

Comment:
:oof: I'm disappointed that `{' '}` is the way to add a space to a string. Seems unnecessary..

Reply:
I'm quite certain that prettier added this...

Reply:
@joshprzybyszewski This seems to be the way to add a space when the line spills over, presumably to make it obvious that there's a space. Here, we've got
```jsx
<p>
  Already have an account?{' '}
  <span>
    ...
  </span>
</p>
```
As opposed to
```jsx
<p>
  Already have an account? <span>...</span>
</p>
```